lib/presentation/theme/theme_manager.dart

```diff
@@ -96,7 +96,7 @@ class ThemeManager {
           const TextStyle(
             fontSize: AppDimensions.d20,
             fontWeight: FontWeight.w700,
-            fontFamily: 'nunitoSans',
+            fontFamily: 'NunitoSans',
```

Comment:
you can move this string to a variable and pass it everywhere so when you change it in the variable it will change everywhere